doc/install/contributing.rst

```diff
@@ -275,6 +275,14 @@ And for building our documentation::
     $ pip install -r requirements_doc.txt
     $ conda install graphviz
 
+.. note::
+   On Windows, if you installed graphviz using the conda command above but still get an error like this::
+
+      WARNING: dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
+
+   you might have to install `graphviz`_ manually.
+   Make sure to agree to add graphviz to path during installation.
```

Comment:
Or it looks like you could also do:
```
export PATH=~/Miniconda3/Library/bin/graphviz:$PATH
```
or the equivalent for your system/env. Not sure if this is better, though.

Reply:
Yes, adding graphviz to path is a much saner solution than a separate install! I'll fix it. 

I'll assume that graphviz has been installed into the conda environment and make use of the `CONDA_PREFIX` variable.

Reply:
fixed by 2bc392ec2c1ad601252660de151b012c07c7ec2c